extensions/amp-timeago/0.2/timeago.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {useResourcesNotify} from '../../../src/preact/utils';
+import {timeago} from '../../../third_party/timeagojs/timeago';
+import { createElement } from 'preact';
+
+/**
+ * @param {!JsonObject} props
+ * @return {Preact.Renderable}
+ */
+export function Timeago(props) {
+  let content;
+  if (props['cutoff']) {
+    const cutoff = parseInt(props['cutoff'], 10);
```

Comment:
This should already be converted to a number via the props definition: https://github.com/ampproject/amphtml/pull/26507/files#diff-a61aeafc22d005aa02e94fa6339f5d88R52